install/tekton-dashboard-deployment.yaml

```diff
@@ -33,3 +33,16 @@ spec:
         env:
         - name: PORT
           value: "9097"
+        # Currently PipelineRuns MUST happen in the same namespace as the install namespace of the Tekton dashboard
+        # the current requirement is that the below namespace values match 
+        - name: PIPELINE_RUN_NAMESPACE
+          value: "replace_me"
+        - name: INSTALLED_NAMESPACE
+          value: "replace_me"
+        # The service account to run PipelineRun pods under
+        # any secrets required for authentication will need to be patched to this service account
+        - name: PIPELINE_RUN_SERVICE_ACCOUNT
+          value: "replace_me"
```

Comment:
Can we add a comment that if this is left blank, the `default` service account will be used? I think that's what this code snippet is for: https://github.com/tektoncd/dashboard/pull/33/files?file-filters%5B%5D=.go&file-filters%5B%5D=.yaml#diff-2a268056787157d73c4168498850de86R244.

Or maybe even leave this value blank by default, because the user does not necessarily need to provide a service account?

Reply:
Yes, I have removed the replace_me and added a comment here to say that default will be used if left blank